### DISCUSSION TITLE
[bug] Is this a bug? "Extra" - averages of the ratings

### DISCUSSION THREAD

**Discussion by lletzgo** [ludumdare/ludumdare#2125] [General]
All of the games submitted via Extra are currently displaying something along the lines of this in their final results:

Overall: N/Ath (0 average from 24 ratings)
Fun: N/Ath (0 average from 24 ratings)
Innovation: N/Ath (0 average from 24 ratings)
Theme: N/Ath (0 average from 24 ratings)
Audio: N/Ath (0 average from 24 ratings)
Humor: N/Ath (0 average from 22 ratings)
Mood: N/Ath (0 average from 22 ratings)

(taken from my own project)

Is that supposed to happen for the averages (as the games aren't meant to take part in the proper rankings), or is that an error? (Me and probably many of the other developers are very curious about our numbers and while comments definitely help get the overall idea, it would still be nice to get to see those.)

**Comment by mikekasprzak** [comment]
Yeah it's not exactly a bug since "showing scores for Extra games" is unfinished, and technically what it's doing is correct (N/A's until finished). I'm also reluctant to hide it since knowing how many ratings per category is still useful to some people.

That said, this is something on my short-list to make Extra better. I'll keep this open for now as a reminder.

**Reply by lletzgo** [reply]
I understand. So the scores will be visible someday, right? (I am talking mostly about the core averages.)

**Reply by mikekasprzak** [reply]
It's going to be a few weeks since I'm taking some time off. Shooting for before the end of May.
